.github/workflows/iroha2-dev-nightly.yml

```diff
@@ -0,0 +1,22 @@
+name: I2::Dev::Nightly::Publish
```

Comment:
If the problem pertains to launching the dispatch, we can commit the file to `main`.

Reply:
Yeah, after this merge I'll PR to `main` those changes for workflows that have `on.workflow_dispatch`